oak_functions/lookup_data_checker/README.md

```diff
@@ -0,0 +1,3 @@
+# Lookup Data Checker
+
+The binary in this directory checks the correctness of the lookup data file.
```

Comment:
perhaps add a code example on how to use it?

Reply:
Done